ValidationApp/src/ValidationApp.cpp

```diff
@@ -29,10 +29,11 @@ ValidationApp::ValidationApp()
 	RegisterScene<DevScene>("DevScene");
 	RegisterScene<LayerScene>("LayerScene");
 	RegisterScene<LevelEditor>("LevelEditor");
+	RegisterScene<ParticleScene>("ParticleScene");
```

Comment:
Voor mergen weer even terugzetten op mainmenuscene is miss handig, voor reviewers kan het tijdelijk op deze scene blijven